### DISCUSSION TITLE
Looking for information on Digit Models

### DISCUSSION THREAD

**Discussion by mherke** [jomjol/AI-on-the-edge-device#2796] [Q&A]
Hi everybody, 

i need some support for the model choice for a gas meter.  So far i understand the difference between dig-class100 vs. dig-cont. 

I have the possibility to choose from the following models:

- dig-class100-0168 s2_q.tflite
- dig-class11_1600 s2.tflite
- dig-class11_1700 s2.tflite
- dig-cont 0611 s3 q.tflite
- dig-cont 0620_s3_q.tflite << currently used

I would now like to know what the difference is between "dig-cont 0611 s3 q.tflite" and "dig-cont 0620_s3_q.tflite", or where I can find more information about all the models. 

best regards and many thanks

Micha

**Comment by caco3** [comment]
Did you check our documentation?

**Reply by caco3** [reply]
https://jomjol.github.io/AI-on-the-edge-device-docs/Choosing-the-Model/

**Reply by mherke** [reply]
Hi, yes i did. But i can not find any information about "dig-cont 0611 s3 q.tflite" and "dig-cont 0620_s3_q.tflite".

**Reply by caco3** [reply]
What do you want to know?

**Reply by doctorkb** [reply]
I have a similar question about the "11" models -- what is difference between 1600 and 1700?

**Reply by schneich** [reply]
Perhaps, it's a stupid question: are these version numbers? The higher, the newer and probably better?
- dig-class11_**1600** s2.tflite
- dig-class11_**1700** s2.tflite
- dig-cont **0611** s3 q.tflite
- dig-cont **0620**_s3_q.tflite

**Reply by SybexX** [reply]
https://jomjol.github.io/AI-on-the-edge-device-docs/Neural-Network-Types/

>The higher, the newer and probably better?

A newer model does not necessarily have to be better, the models are not tested with all types of water meters/electricity meters/gas meters...., so it may also be that an older model works better for you, so you should always try out several models and choose the best one for you.
